### ISSUE TITLE
Newly manufactured (?) NZXT Kraken X63 not working in FanControl

### ISSUE THREAD

**Issue by Fraser-Watson** [Rem0o/FanControl.Releases#1561]
I just got a replacement NZXT Kraken X63 from NZXT as my pump was louder than it should be above 60%, but when I've installed the replacement, I get no control over it within FanControl, and also no water temp/pump RPM sensor in HWiNFO64. Has the communication protocol changed? It is controllable within CAM but not in FanControl. Firmware versions are the same (V2.1) between the two coolers.

**Comment by Rem0o** [comment]
Stupid question: have you actually plugged everything the same?  Especially the usb cable.

**Comment by Fraser-Watson** [comment]
> Stupid question: have you actually plugged everything the same? Especially the usb cable.

Yes, everything is plugged in. I can control it via CAM and LiquidCTL so I know it’s plugged in correctly

**Comment by supercarkiller** [comment]
I am having the similar  issue with the same pump. The pump speed shows in the "Speed" Section however I am unable to pair it with "Controls" Section.

**Comment by supercarkiller** [comment]
After going into sensors settings and selecting (Controller (various third parties)) 
a new Speed Card showed in the UI for NZXT AIO Pump and synced with Controls Card with no issues!

However I now have two Speed Cards..
The card that was able to sync rpm seems to be different from the second speed cards rpm that I'm not able to sync with. It a little confusing and I don't know where the second speed card is from.

**Comment by Fraser-Watson** [comment]
> 

Hi,

If you have access to the AIO Pump card, then your issue is unrelated to the one originally raised by me. Please do not hijack issues, instead, create your own issue. You will probably have more help this way and prevents the thread going off-topic.

**Comment by brokenmass** [comment]
@Rem0o this should be fixed in latest fancontrol release as librehardwaremonitor now support x73 new pid and new krakens
